backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -300,22 +314,36 @@ public static Observation convertToObservation(
       String diseaseCode,
       String diseaseName,
       String resultCode,
-      boolean corrected,
+      TestCorrectionStatus correctionStatus,
       String correctionReason,
       String id) {
     var observation = new Observation();
     observation.setId(id);
-    observation.setStatus(ObservationStatus.FINAL);
+    setStatus(observation, correctionStatus);
     addCode(diseaseCode, diseaseName, observation);
     addValue(resultCode, observation);
-    addCorrectionNote(corrected, correctionReason, observation);
+    addCorrectionNote(
+        correctionStatus != TestCorrectionStatus.ORIGINAL, correctionReason, observation);
     return observation;
   }
 
+  private static void setStatus(Observation observation, TestCorrectionStatus correctionStatus) {
+    switch (correctionStatus) {
+      case ORIGINAL:
+        observation.setStatus(ObservationStatus.FINAL);
+        break;
+      case CORRECTED:
+        observation.setStatus(ObservationStatus.CORRECTED);
+        break;
+      case REMOVED:
+        observation.setStatus(ObservationStatus.ENTEREDINERROR);
```

Comment:
Should this be ENTEREDINERROR or CANCELLED? I notice it's cancelled down in DiagnosticReportStatus, not sure how important it is to be consistent across all statuses.

Reply:
I think it depends, so for `ENTEREDINERROR` it says `The observation has been withdrawn following previous final release. This electronic record should never have existed ` so that makes me think if it's ever been reported as `FINAL` it should not be reported as cancelled. I think that's a mistake in DiagnosticReport then I can fix that in another PR.

Reply:
Sounds good to me, I agree that ENTEREDINERROR seems like the right status here. Addressing that in a separate PR works 👍